### PR TITLE
MDL2 updates for SearchBox and CommandBar

### PR DESCRIPTION
**Pull request by wdo3650**

### CODE REVIEW

File: src/documentation/pages/SearchBox/examples/SearchBoxExampleJS.hbs [hunk 1]
```diff
@@ -1,3 +1,8 @@
+<style>
+  body {
+    background-color: #EFEFEF;
```
Comment (mikewheaton) [inline src/documentation/pages/SearchBox/examples/SearchBoxExampleJS.hbs]: Does the SearchBox always have to be displayed on a grey background?

**Comment by mikewheaton** [comment]
Approved with a minor comment. Built and tested locally and it looks/works great!

[Approved with PullApprove](https://pullapprove.com/OfficeDev/Office-UI-Fabric/pull-request/692/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OfficeDev/Office-UI-Fabric)

**Comment by wdo3650** [comment]
@mikewheaton good catch. I put that in for testing and forgot to take that out.

**Comment by Jahnp** [comment]
Approved! Looks great.

[Approved with PullApprove](https://pullapprove.com/OfficeDev/Office-UI-Fabric/pull-request/692/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OfficeDev/Office-UI-Fabric)

**Pull request merged**